Source/WebCore/rendering/mac/RenderThemeMac.mm

```diff
@@ -1015,9 +1015,16 @@ static void setFontFromControlSize(RenderStyle& style, NSControlSize controlSize
     return sizes;
 }
 
-void RenderThemeMac::adjustMenuListStyle(RenderStyle& style, const Element* e) const
+void RenderThemeMac::adjustMenuListStyle(RenderStyle& style, const Element* element) const
 {
-    RenderThemeCocoa::adjustMenuListStyle(style, e);
```

Comment:
I think there is a subtle behavior change here. Previously, when `vectorBasedControlsOnMacEnabled` is disabled, this would call into `RenderTheme::adjustMenuListStyle`.

We've now removed that call entirely in that path.